notebooks/00_quick_start/run_notebook_on_azureml.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>i think we should merge <a href="https://github.com/Microsoft/Recommenders/pull/728" target="_blank">https://github.com/Microsoft/Recommenders/pull/728</a> first and then use the get_or_create_workspace function there to minimize use of scripts post-setup</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I'm hoping this PR can make it into master with #720 . If #728 can get in with, then I will rebase to use get_or_create_workspace

Reply:
#728 is merged to staging

Reply:
rebased to latest staging